tests/unit/internal-session-test.js

```diff
@@ -18,7 +18,8 @@ describe('InternalSession', () => {
     store         = EphemeralStore.create();
     authenticator = Authenticator.create();
     session       = InternalSession.create({ store, container });
-    sinon.stub(container, 'lookup').withArgs('authenticator').returns(authenticator);
+    sinon.stub(container, 'lookup').withArgs('authenticator').returns(authenticator)
+      .withArgs('authenticator:test').returns(authenticator);
```

Comment:
From my diving into this, either the specs within `tests/acceptance/authentication-test.js` or the methods within `test-support/helpers/ember-simple-auth.js` seem to be the culprit for not properly cleaning the session store. I found this because commenting out the acceptance tests caused the two failing specs to pass.

I [opened a PR](https://github.com/jcope2013/ember-simple-auth/pull/2) to resolve it. I'm not sure if my commit is the best approach to resolve it, but it does seem to fix the specs without improperly stubbing for `'authenticator:test'`.